Makefile

```diff
@@ -62,6 +62,11 @@ lint:
 release-major release-minor release-patch:
 	@$(XYZ) --increment $(@:release-%=%)
 
+.PHONY: bundle
+bundle:
+	mkdir dist || echo "Directory already exists"
+	browserify index.js > dist/sanctuary.js
```

Comment:
Is Browserify necessary? If so, why is it necessary?

Reply:
It means you include one script tag, and sanctuary-def, etc are all bundled together.  I think the main reason you'd want this is for demoing/testing out the library with minimal friction.

Reply:
I'm certainly in favour of providing a convenient `sanctuary+dependencies` bundle. It seems we're using Browserify here for dependency resolution. Is this correct? If so, I'm hoping something simpler would suffice:

```
cat \
  node_modules/sanctuary-type-identifiers/index.js \
  node_modules/sanctuary-type-classes/index.js \
  node_modules/sanctuary-def/index.js \
  index.js \
  >dist/sanctuary.js
```

Reply:
Yeah that looks good.